tests/Integration/Utility/CartOld.php

```diff
@@ -36,6 +36,8 @@
 use Tools;
 
 /**
+ * @todo: should this class be deleted already? (or at least deprecated?) It doesn't seem to be used anymore
```

Comment:
note this cart old should be deprecated and later removed, no? I don't see it being used anywhere else except the tests anymore

Reply:
Actually it seems like it's still used here https://github.com/PrestaShop/PrestaShop/blob/edd1360cd80b6ea09872b48267cda183fff007a6/tests/Integration/Behaviour/Features/Context/CartFeatureContext.php#L82

So I guess we should keep it for now, and you can remove your todo comment, it was used in the tests to compare both computing methods for a while:

https://github.com/PrestaShop/PrestaShop/blob/edd1360cd80b6ea09872b48267cda183fff007a6/tests/Integration/Behaviour/Features/Context/CartFeatureContext.php#L164-L173

Reply:
There are still many usage in behat of this, unless you cleaned them all in this PR? Maybe you're right and it's worth cleaning this but I'm not sure it's the right PR for that